bigbluebutton-html5/imports/startup/server/redis.js

```diff
@@ -106,8 +106,18 @@ class RedisPubSub {
 
     this.didSendRequestEvent = false;
     const redisHost = process.env.REDIS_HOST || Meteor.settings.private.redis.host;
-    this.pub = Redis.createClient(Meteor.settings.private.redis.port, redisHost);
-    this.sub = Redis.createClient(Meteor.settings.private.redis.port, redisHost);
+    const redisConfig = {
+      port: Meteor.settings.private.redis.port,
```

Comment:
We can group the parameters - something like

```
const redisConf = Meteor.settings.private.redis;
const { host, password, port } = redisConf;
```

`host` was used a few lines above.
For the password is it better to have it as `null` or `""`

Reply:
The default value for password is null, therefore I went with that in this situation.